src/Aspire.Dashboard/Resources/Metrics.resx

```diff
@@ -170,9 +170,9 @@
     <value>Instrument</value>
   </data>
   <data name="DashpagesSectionName" xml:space="preserve">
-    <value>Dashpages</value>
+    <value>Highlights</value>
```

Comment:
Is that a new name? When we settle on a name I think APIs and files should be changed to reflect the new name.

Reply:
Updates files/APIs to "Highlight"